### PR TITLE
Added tests to validate all student submissions

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
fixes https://github.com/mitodl/edx-sga/issues/232

#### What's this PR do?
- Add multiple test to validate that file content of students so that student uploads could not overwrite each other. Tests for PR https://github.com/mitodl/edx-sga/pull/230

#### How should this be manually tested?
Run tests on edX platform

#### Where should the reviewer start?
run 
- python manage.py lms --settings=test test edx_sga.tests.integration_tests.StaffGradedAssignmentXblockTests.test_staff_download
- python manage.py lms --settings=test test edx_sga.tests.integration_tests.StaffGradedAssignmentXblockTests.test_download_annotated 

@pdpinch

### CODE REVIEW

**Comment by noisecapella** [comment]
If I run these updated tests against commit 74e0b3491cbc50408081655131695bb74eb48399 I don't see any tests fail. If the tests are asserting that #230 fixes the file content duplication issue then they should fail when run against a commit without the fix

**Comment by amir-qayyum-khan** [comment]
@noisecapella you are testing this PR against wrong hash.
The error was introduce with my PR https://github.com/mitodl/edx-sga/pull/226.
Please test it against it.

Thanks,

File: edx_sga/tests/integration_tests.py [hunk 1]
```diff
@@ -139,52 +140,53 @@ def make_student(self, block, name, make_state=True, **state):
...
 
-        user = User(username=name)
-        user.save()
-        profile = UserProfile(user=user, name=name)
-        profile.save()
-        if make_state:
-            module = StudentModule(
-                module_state_key=block.location,
-                student=user,
+        with transaction.atomic():
```
Comment (noisecapella) [inline edx_sga/tests/integration_tests.py]: What's the purpose of this? All tests already run within a transaction

File: edx_sga/tests/integration_tests.py [hunk 2]
```diff
@@ -509,32 +520,44 @@ def test_download_annotated(self):
 
     def test_staff_download(self):
         """
-        Test download for staff.
+        Test download for staff. validate each student assignment
```
Comment (noisecapella) [inline edx_sga/tests/integration_tests.py]: Can you move the period to the end? This should read `Test download for staff and validate each student assignment.`

File: edx_sga/tests/integration_tests.py (hunk 1 above)
Comment (amir-qayyum-khan) [reply edx_sga/tests/integration_tests.py]: making tow students in loop without this throws error that block should atomic

**Comment by amir-qayyum-khan** [comment]
@noisecapella any update

**Pull request merged**